packages/gatsby-theme-store/src/components/ToastProvider/index.tsx

```diff
@@ -0,0 +1,20 @@
+// snippet of code to use ToastProvider lazy
+
+// import React, { Suspense, lazy } from 'react'
+// import type { FC } from 'react'
+
+// const loader = () => import('@vtex/store-ui/src/ToastMessage/ToastProvider')
+
+// const ToastProvider = lazy(loader)
+
+// export const Provider: FC = ({ children }) => {
+//   return (
+//     <Suspense fallback={null}>
+//       <ToastProvider>{children}</ToastProvider>
+//     </Suspense>
+//   )
+// }
+
+import type { PropsWithChildren } from 'react'
+
+export const Provider = ({ children }: PropsWithChildren<null>) => children
```

Comment:
what is it?

Reply:
Is a Noop component, because we don't enforce all stores has ToastProvider.